tools/function_finder/function_finder_psx.py

```diff
@@ -180,8 +209,8 @@ def find_wip(o):
 
     # delete asm text
     for o in output:
-        del o[7:]
+        del o[8:]
 
-    headers = ["Ovl", "Function", "Length", "Branches", "Jtbl", "WIP", "%"]
+    headers = ["Ovl", "Function", "Length", "Branches", "Jtbl", f"[Duplicate]({base_url}/duplicates.txt)", "WIP", "%"]
```

Comment:
I made the header a link to the base list but this might be redundant. Perhaps this could link to https://github.com/Xeeynamo/sotn-decomp/wiki/Decompilation#duplicate-functions instead